$NetBSD: patch-LiteEditor_CMakeLists.txt,v 1.1 2016/06/15 22:04:17 kamil Exp $

Add switch for NetBSD to not fall under "other OS".

Fix path of installation of manpages.

--- LiteEditor/CMakeLists.txt.orig	2019-05-21 17:52:21.000000000 +0300
+++ LiteEditor/CMakeLists.txt
@@ -46,6 +46,8 @@ set( ADDITIONAL_LIBRARIES "" )
 if (UNIX)
     if ( IS_FREEBSD )
         set(ADDITIONAL_LIBRARIES "-lkvm")
+    elseif ( IS_NETBSD )
+       set(ADDTIONAL_LIBRARIES "-lutil")
     elseif ( UNIX AND NOT APPLE )
         set(ADDITIONAL_LIBRARIES "-ldl -lutil -lgobject-2.0")
     else ( )
